datasets/eu/reg_2878_sanctions/crawler.py

```diff
@@ -0,0 +1,53 @@
+"""
+Add supplemental list of EU sanctions from annex in Regulation  2023/2878
+"""
+
+import csv
+from typing import Dict, Iterable
+
+import zavod.helpers as h
+from zavod import Context
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    """Process one row of the CSV data"""
+    row_id = row.pop("List ID").strip(" \t.")
+    entity_type = row.pop("Type").strip()
+    name = row.pop("Name").strip()
+    country = row.pop("Country").strip()
+
+    # FIXME: Not sure of the correct way to automate this with
+    # lookups, is that documented anywhere? (using the "companies" key
+    # gives an exception that cannot be understood)
+    if name == "LLC CST":
+        name = "Zala Aero Group"
```

Comment:
I think in this case fixing in the spreadsheet made most sense. Matching on `Zala Aero Group` in the type.country lookup and returning Russia wouldn't have been ideal either, since Russia isn't actually mentioned in there.

Since this is manually extracted, the best fix was in the spreadsheet.

If we were scraping the data programmatically, I might write the code in such a way that my regex doesn't match on two sets of parens. So when the regex matches, I'd use the captured groups. If it doesn't match, I'd put the whole unmatched text into [context.lookup](https://github.com/opensanctions/opensanctions/blob/main/datasets/fr/tresor/crawler.py#L54) and [manually split the values I want from it in lookups](https://github.com/opensanctions/opensanctions/blob/main/datasets/fr/tresor/fr_tresor_gels_avoir.yml#L59)
```suggestion
```